easybuild/easyconfigs/p/pydantic/pydantic-1.10.13-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
```

Comment:
This should be using `CargoPythonBundle` instead, because it involves building a bunch of Rust crates.

See also #19716

Reply:
@jfgrimm Can you update your PR with current `develop` to use the `pydantic` easyconfig that was merged just now in https://github.com/easybuilders/easybuild-easyconfigs/pull/19716?

Reply:
wait, this explicitly needs `pydantic < 2`

Reply:
Which package requires `pydantic < 2` This would be an issue as we need the 2.x in other software and 12.2.0 already has a 2.x
napari seems to be fine: https://github.com/napari/napari/blob/0be7f18fb67d4f68876a0e5ca04c619dd343f456/setup.cfg#L67

Reply:
https://github.com/napari/napari/blob/v0.4.18/setup.cfg#L67

Reply:
pydantic 2 compatibility was only added in https://github.com/napari/napari/commit/e9ea8d06d5f9f8e029f464b3938e6a60df3f6bb8

Reply:
That will be a major issue I guess. To me it looks like that commit you linked adds forward compatibility to napari, i.e. it will work with pydantic 2 and 1.

If that applies more or less cleanly to 0.4.18 I'd suggest to include this as a patch instead of introducing a 2nd pydantic version into this toolchain

Reply:
sure, I can try that

Reply:
good should, only needed minor tweaks